tools/tuf/releaser.sh

```diff
@@ -144,7 +144,7 @@ release_fleetd_to_edge () {
     DESKTOP_ARTIFACT_DOWNLOAD_DIRECTORY="$ARTIFACTS_DOWNLOAD_DIRECTORY/desktop"
     mkdir -p "$DESKTOP_ARTIFACT_DOWNLOAD_DIRECTORY"
     "$GO_TOOLS_DIRECTORY/download-artifacts" desktop \
-        --git-branch "$BRANCH_NAME" \
+        --git-tag "$ORBIT_TAG" \
```

Comment:
@sgress454 Just this minor fix because we are not relying on a branch/PR to trigger 
Fleet Desktop builds anymore. (We now use the orbit tag)

Reply:
good catch!

Reply:
Wasn't a catch :) `releaser.sh` was failing when attempting to release fleetd 1.41.0 to edge.